test/unit/test_annotated_code.c

```diff
@@ -0,0 +1,298 @@
+#include <r_util.h>
+#include <r_vector.h>
+#include <r_core.h>
+#include <r_cons.h>
+#include <r_util/r_annotated_code.h>
+
+#include "minunit.h"
+
+static RCodeAnnotation makeRCodeAnnotation(int st, int en, RCodeAnnotationType typec, 
+											ut64 offset, RSyntaxHighlightType types){
```

Comment:
Excessive indent, you can put the entire signature in one line. Also please use `snake_case` for function names, i.e. `make_code_annotation` for example (R not needed imo).